lib/ruby_lsp/ruby_lsp_rails/hover.rb

```diff
@@ -25,21 +25,21 @@ class Hover
           client: RunnerClient,
           response_builder: ResponseBuilders::Hover,
           nesting: T::Array[String],
-          index: RubyIndexer::Index,
+          global_state: GlobalState,
           dispatcher: Prism::Dispatcher,
         ).void
       end
-      def initialize(client, response_builder, nesting, index, dispatcher)
+      def initialize(client, response_builder, nesting, global_state, dispatcher)
         @client = client
         @response_builder = response_builder
         @nesting = nesting
-        @index = index
+        @global_state = global_state
```

Comment:
Let's put the index in a variable to avoid doing `@global_state.index` multiple times.